code/modules/mob/living/silicon/ai/freelook/eye.dm

```diff
@@ -21,6 +21,7 @@
 /mob/camera/aiEye/Initialize()
 	. = ..()
 	GLOB.aiEyes += src
+	icon_state = "ai_camera" + (GLOB.aiEyes.len % 3) // Yogs -- multicoloured AI eyes

```

Comment:
Won't this always choose the 0th sprite? I think you a rand there.

Reply:
It chooses the 0th, 1st, or 2nd sprite, depending on the number of AIs. The first AI would always get the first one, since ``(1 % 3) == 1``.

Although, yeah, the way I'm doing it here is still a bit wonky. Will work on it.

Reply:
oh, derp, I misunderstood this